### PR TITLE
Add PullRequestSerialized

### LINKED ISSUE #859: Add pullrequest collection to the database
Since the aim is for Repairnator to create pull requests automcatically, it would be useful if we could easily get information about all pull requests it has created. Since we already have a database, the easiest way of doing this would be through a new collection in the database which we can quarry in much the same way as others.

### PR DESCRIPTION
**Pull request by btellstrom**
Adds a new class which serialized pull request so that we may easily extract data about the number of pull-requests Repairnator has created.

Closes #859

### CODE REVIEW

**Comment by btellstrom** [comment]
Oh right, the CI will fail here due to #860 but there are no new tests (might want to add them, but did not find any existing ones of a similar sort).

File: repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/serializer/PullRequestSerializer.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import com.google.gson.JsonObject;
+
+import fr.inria.jtravis.entities.Build;
+import fr.inria.spirals.repairnator.process.inspectors.ProjectInspector;
+import fr.inria.spirals.repairnator.serializer.engines.SerializedData;
+import fr.inria.spirals.repairnator.serializer.engines.SerializerEngine;
+import fr.inria.spirals.repairnator.utils.DateUtils;
+
+/**
+ *
```
Comment (monperrus) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/serializer/PullRequestSerializer.java]: add two sentences of explanation? when one uses this? for which goal?

**Comment by monperrus** [comment]
ci still failing?

**Comment by btellstrom** [comment]
Both this and #864 fail due to log length, but they should both have been merged with master, so I am really not sure what is the issue here...

**Pull request merged**